lib/.eslintrc.yaml

```diff
@@ -80,7 +82,9 @@ rules:
     - name: Uint8ClampedArray
     - name: URIError
     - name: WeakMap
+      into: Safe
     - name: WeakSet
+      into: Safe
```

Comment:
Can we add Safe prefix for Promise as well?
https://github.com/nodejs/node/blob/adae822035b640531044b33892c32a3a3011cf5b/lib/internal/per_context/primordials.js#L109

Reply:
I'm planing to open a PR as soon as https://github.com/nodejs/node/pull/36019 and https://github.com/nodejs/node/pull/36023 land to retire `SafePromise`: it's barely used in node core, and we can't enforce it anyway – unless we forbid the use of `async`/`await` in core.

Reply:
async/await is Safe anyway ^^

Reply:
Well, `await` is safe in the sense that it doesn’t leak the internal state of the function, unlike a `.then(…)` callback chain.

---

But it’s not safe in the sense that if user code were to break the `Promise.prototype.constructor` property (by turning it into a throwing accessor) or the `Promise.prototype.then` method, then `await` will break, unlike `SafePromise`.

## Spec:
- <https://tc39.es/ecma262/#await>
- <https://tc39.es/ecma262/#sec-promise-resolve>

Reply:
@aduh95 I got it. I don't think it's a problem to separate the prefix of Promise into another PR.